els-front/src/components/ProtectedRoute.js

```diff
@@ -0,0 +1,16 @@
+import {
+    Navigate,
+  } from 'react-router-dom';
+  
+const ProtectedRoute = ({ user, redirectPath = '/',for_auth = false,children }) => {
```

Comment:
```suggestion
const ProtectedRoute = ({ user, redirectPath = '/', for_auth = false, children }) => {
```

- spaces
- add new line at EOF, also apply to other files